plugins/objdump/objdump_main.ml

```diff
@@ -77,8 +76,8 @@ let main () =
   Stream.merge Project.Info.arch Project.Info.file ~f:run_objdump |>
   Symbolizer.Factory.register name
 
-let info =
-  let man = [
+let () =
+  Config.manpage [
```

Comment:
This was still under discussion at the time, and we hadn't decided in either direction. I have not changed it in this PR yet, since we are still uncertain about it (as per our offline discussion)